src/AnnotatedRoutes/tests/IntegrationTest.php

```diff
@@ -63,7 +66,10 @@ public function get(
         array $headers = [],
         array $cookies = []
     ): ResponseInterface {
-        return $this->app->getHttp()->handle($this->request($uri, 'GET', $query, $headers, $cookies));
+        return $this->app->getContainer()->runScope(
+            new Scope(ScopeName::Http),
+            fn (Http $http) => $http->handle($this->request($uri, 'GET', $query, $headers, $cookies))
```

Comment:
i wonder if we can have container in HTTP scope already here... @roxblnfk like $this->getHttpScopeContainer() and then we work from there. can we do that?

Reply:
I think about using attributes for scoped tests:

```php
#[TestScope('http.request')]
public function testSomethingInHttpRequest() {...}
```

Reply:
@wolfy-j @roxblnfk Added **TestScope** attribute to the spiral/testing package, simplified tests.